src/components/Attachment/index.ts

```diff
@@ -184,7 +184,9 @@ class Attachment extends BridgeBase {
       fileTemp = attachments[0];
 
       if (fileTemp.responseProps) {
+        // @ts-ignore - Property 'attachmentsInfo' does not exist on type 'C11nEnv'
```

Comment:
If we want to extend C11nEnv by adding attachmentsInfo object then we should create a type locally that extends C11nEnv with attachmentsInfo object.

Reply:
We are currently accessing thePConn object in the attachment component from the Bridgebase component. However, this attachmentInfo will be removed when we implement support for multiple attachments in the WC SDK.